src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,64 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  getMovies: (movies: Movie[]) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ getMovies }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState<string>('');
```

Comment:
don't need typing here, it is automatically applied with a default value
```suggestion
  const [query, setQuery] = useState('');
```